pkg/wrapper/nmt_wrapper.go

```diff
@@ -103,3 +109,14 @@ func (w *ErasuredNamespacedMerkleTree) Prove(ind int) (nmt.Proof, error) {
 func (w *ErasuredNamespacedMerkleTree) Tree() *nmt.NamespacedMerkleTree {
 	return w.tree
 }
+
+// incrementShareIndex increments the share index by one.
+func (w *ErasuredNamespacedMerkleTree) incrementShareIndex() {
+	w.shareIndex++
+}
+
+// isQuadrant0 returns true if the current share index and axis index are both
+// in the original data square.
+func (w *ErasuredNamespacedMerkleTree) isQuadrant0() bool {
+	return !(uint64(w.shareIndex)+1 > w.squareSize || uint64(w.axisIndex)+1 > w.squareSize)
+}
```

Comment:
[very optional] removed a negation because this may be clearer for readers

```suggestion
// isQuadrantZero returns true if the current share index and axis index are both
// in the original data square.
func (w *ErasuredNamespacedMerkleTree) isQuadrantZero() bool {
	return w.shareIndex < w.squareSize && w.axisIndex < w.squareSize
}
```

Reply:
yes good idea, I just copy pasta'd

Reply:
https://github.com/celestiaorg/celestia-app/pull/1418/commits/68a1ea6d659e99658ef33f14f9a6a5ef4e37595a